package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "terminal-pedal-picker",
+  "version": "1.0.0",
+  "description": "## Description",
```

Comment:
Either delete this line or change it to an actual description of the project. Do not use markdown in any of the `package.json` fields.